GETTING_STARTED.md

```diff
@@ -0,0 +1,166 @@
+
+## Initialising your repository
+
+Firstly, we're going to need a repository to build our monorepo in. Running the following commands in your terminal will get you a new repository ready for work:
+
+```shell
+mkdir my-new-monorepo
+cd my-new-monorepo
+git init
+git add .
+git commit -m "look, our new monorepo!"
+```
+
+From here on out, unless otherwise stated, all terminal commands are assumed to be run from this root folder. We recommend you open this project now in an editory [such as VS Code](https://code.visualstudio.com/) so you can explore your monorepo as it comes together (we will be using screenshots from our own code editor to help demonstrate structures)
+
+Next, we want to add our `package.json`. Create the file, then copy the following into it:
+
+```json
+{
+  "name": "@monorepo-starter/root",
+  "version": "1.0.0",
+  "private": true
+}
+```
+
+The root `package.json` of your monorepo should always be private, to avoid accidentally publishing it.
+
+## Setting up your packages
+
+The next thing we want to do is define where our packages live. We do this by adding a `workspaces` field to our root `package.json`. For our project, we are going to say that packages can live in three locations:
+
+- A `/apps` folder for blah blah blah
+- A `/scripts` folder for blah blah blah
+- A `/packages` folder for... packages...
+
+We only want to allow folders at the first level of each of these to be checked for being packages, so we want a workspaces field that looks like:
+
+```json
+  "workspaces": [
+    "packages/*",
+    "apps/*",
+    "scripts/*"
+  ]
+```
+
+Which should leave our root `package.json` looking like:
+
+```json
+{
+  "name": "@monorepo-starter/root",
+  "version": "1.0.0",
+  "private": true,
+  "workspaces": ["packages/*", "apps/*", "scripts/*"]
+}
+```
+
+With that set up, you can now run `yarn` as your install script, and it will install all dependencies of all packages within those folders.
+
+Next up, let's add some packages so we can have some dependencies!
+
+### Your package folders
+
+For this tutorial, we are going to be setting up 5 packages. They will be:
+
+- `@monorepo-starter/simple-package`
+- `@monorepo-starter/private-package`
+- `@monorepo-starter/with-multiple-entrypoints`
+- `@monorepo-starter/placeholder-website`
+- `@monorepo-starter/placeholder-app`
+
+We are going to learn something different while setting up each of these packages. We will set up the `simple-package`, and `placeholder-website` now, and `with-multiple-entrypoints`, and `placeholder-app` a little later when we set up our build tooling. Finally, we will add our `private-package` when we set up publishing.
+
+#### Adding `@monorepo-starter/simple-package`
+
+This package is going to be our bedrock simplest package.
+
+```shell
+mkdir packages
+mkdir packages/simple-package
+```
+
+In our newly created `simple-package` directory, add the following `package.json`:
+
+```json
+{
+  "name": "@monorepo-starter/simple-package",
```

Comment:
I would like if you explain the `@` syntax

Reply:
It's pseudo-covered by [this part of the style guide](https://github.com/thinkmill/monorepo#one-scope-for-the-monorepo-all-packages-are-scoped) but I have made an assumption that everyone knows and understands what a scoped package is when they see it. Do you think this is not true?